### ISSUE TITLE
Helm v3 - lint error messages contain a malformed string

### ISSUE THREAD

**Issue by andytom** [helm/helm#6246]
Output of `helm version`:
```
$ helm3 version
version.BuildInfo{Version:"v3.0.0-alpha.2", GitCommit:"97e7461e41455e58d89b4d7d192fed5352001d44", GitTreeState:"clean", GoVersion:"go1.12.7"}
```

Output of `kubectl version`:
N/A

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
N/A

When linting a chart if there are any errors the message printed out contains a malformed string.

Steps to reproduce:
```
$ mkdir /tmp/broken_chart
$ touch /tmp/broken_chart/Chart.yaml
$ helm3 lint /tmp/broken_chart
Error: 1 chart(s) linted, 1 chart(s) failed
        %!s(<nil>)
        [ERROR] Chart.yaml: name is required
        [ERROR] Chart.yaml: directory name (broken_chart) and chart name () must be the same
        [ERROR] Chart.yaml: apiVersion is required. The value must be either "v1" or "v2"
        [ERROR] Chart.yaml: version is required
        [INFO] Chart.yaml: icon is recommended
        [INFO] values.yaml: file does not exist
        [WARNING] templates/: directory not found
```

**Comment by andytom** [comment]
I think the problem might be in https://github.com/helm/helm/blob/dev-v3/pkg/action/lint.go#L70

The value of `err` is `nil` and we are appending this `nil` to the returned `result.Errors` however I'm not sure what the expected behaviour is here.

**Comment by andytom** [comment]
Looks like this was fixed as part of #6241.
